lms/templates/registration/password_reset_complete.html

```diff
@@ -45,9 +45,7 @@ <h1 class="title">
   <section class="passwordreset container">
     {% block content %}
     <section role="main" class="content">
-      {% blocktrans with link_start='<a href="/login">' link_end='</a>' %}
-      Your password has been set.  You may go ahead and {{ link_start }}log in{{ link_end }} now.
-      {% endblocktrans %}
+      {% trans "Your password has been changed successfully. You are now logged in." %}
```

Comment:
We don't want to change the first part of the message,
`Your password has been set.`;
we should only be changing the second,
`s/You may go ahead and {{ link_start }}log in{{ link_end }} now./You are now logged in.`.

1. This is what the story requested.
2. This minimizes the fork from upstream.